charts/jaeger-v2/templates/deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jaeger
+  labels:
+    app: jaeger
+    namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: jaeger
+  template:
+    metadata:
+      labels:
+        app: jaeger
+    spec:
+      containers:
+        - name: jaeger
+          image: jaegertracing/jaeger:latest
+          args: 
+            - "--config"
+            - "/etc/jaeger/config.yaml"
+          ports:
+            - containerPort: 16686
+              name: ui
+            - containerPort: 14268
+              name: collector
+            - containerPort: 14250
+              name: grpc
+            - containerPort: 9411
+              name: zipkin
+          env:
+            - name: COLLECTOR_ZIPKIN_HOST_PORT
+              value: ":9411"
```

Comment:
This has no meaning for v2

```suggestion
```

Reply:
should i also remove 
```
- containerPort: 9411
              name: zipkin
```

Reply:
no, port mapping can stay, we still run zipkin receiver, it just doesn't need any env vars